packages/remix-dev/compiler/compiler.ts

```diff
@@ -96,6 +100,7 @@ export let create = async (ctx: Context): Promise<Compiler> => {
     let js = await tasks.js;
     if (!js.ok) throw error ?? js.error;
     let { metafile, hmr } = js.value;
+    await createBrowserMetaFile(metafile)
```

Comment:
Conditional should be visible at this level:

```suggestion
    if (ctx.config.metafile) {
      await writeMetafile( ctx, "js", metafile)
    }
```

Reply:
All right, I'll take care of it later.